Vagrantfile

```diff
@@ -7,8 +7,8 @@ Vagrant.require_version '>= 1.8.0'
 
 cwd = File.dirname(File.expand_path(__FILE__))
 config_dir = '.beetbox/'
-vagrant_config = "#{cwd}/#{config_dir}config.yml"
-local_config = "#{cwd}/#{config_dir}local.config.yml"
+project_config = "#{dir}/#{config_dir}config.yml"
```

Comment:
Regression error, changing back to `dir` which is not set will cause this Vagrantfile to break.